io.github.verticalsync.sunroof.yml

```diff
@@ -0,0 +1,72 @@
+app-id: io.github.verticalsync.sunroof
+
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+
+command: startsunroof
+separate-locales: false
+
+finish-args:
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.kde.StatusNotifierWatcher # Tray functionalities on KDE
+  - --device=all # Needed for GPU acceleration and the webcam
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-download # This and the above two are used for drag-n-drop, and download managing
+  - --filesystem=xdg-run/pipewire-0 # Pipewire interfacing
+  - --filesystem=~/.steam # Needed for SteamOS integration, as the XDG OpenURL portal doesn't work properly in Game Mode. We only need ~/.steam/steam.pipe but Flatpak can't correctly mount just that: 'File "/home/deck/.steam/steam.pipe" has unsupported type 0o10000'
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons # This is used to show the correct cursor on Wayland
+
+modules:
+  - name: Sunroof
+    buildsystem: simple
+    build-commands:
+      - chmod +x Sunroof.AppImage
+      - ./Sunroof.AppImage --appimage-extract
+      - desktop-file-edit --set-key="Exec" --set-value="startsunroof" --set-icon=$FLATPAK_ID
+        squashfs-root/sunroof.desktop
+      - install -D squashfs-root/usr/share/icons/hicolor/256x256/apps/sunroof.png
+        /app/share/icons/hicolor/256x256/apps/io.github.verticalsync.sunroof.png
+      - install -D squashfs-root/usr/share/icons/hicolor/16x16/apps/sunroof.png /app/share/icons/hicolor/16x16/apps/io.github.verticalsync.sunroof.png
+      - install -D squashfs-root/sunroof.desktop /app/share/applications/io.github.verticalsync.sunroof.desktop
+      - install -Dm755 startsunroof /app/bin/startsunroof
+      - mv squashfs-root /app/bin/sunroof
+
+    sources:
+      - type: file
+        path: startsunroof
+
+      - type: file
+        url: https://raw.githubusercontent.com/verticalsync/Sunroof/master/meta/me.verticalsync.Sunroof.metainfo.xml
```

Comment:
the name MUST match the app-id. It's case sensitive. Also even with the wrong name it is name installed.